### DISCUSSION TITLE
Extensão não funciona no Chrome

### DISCUSSION THREAD

**Discussion by MixieAppBr** [aclap-dev/video-downloadhelper#1442] [License & Purchase requests]
### What's your issue?

Sou do Brasil e fiz a compra da extensão para o Chrome mas quando fui efetuar o download ele fala que o Chorme não suporta o download para videos e que eu deveria utilizar a extensão para o Firefox. Então comprei a premium e não posso usar para o que desejo. Existe alguma forma de reeembolso ou a troca da minha licensa para o firefox ? 

meu email de cadastro é mixieappbr@gmail.com e o começo da minha chave é ....

### The 10 first digits (only 10) of your license number or email address

0c3b74006b

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
